### DISCUSSION TITLE
copy labels from the scanned resource

### DISCUSSION THREAD

**Discussion by mycodeself** [aquasecurity/trivy-operator#124] [Development]
Hi, 

I would like to know what you think of an idea that about a use case of the operator.

A good practice is to have label conventions on the different kubernetes resources, these labels can be used to group different resources. When the operator generates a report for a specific resource I would like to have the possibility to copy some labels (which could be parameterisable) from the scanned resource to the report.

This way we could easily get all the reports of a resource grouping from certain tags. A scenario would be where for example the different workloads are labelled by the different teams with a `team` label.

I hope I have explained myself and I would like to know what you think!

**Comment by erikgb** [comment]
I support this, and I even think we could just copy all labels from the owning resource. Making it parametrizable would just introduce unnecessary complexity IMO.

**Reply by louzadod** [reply]
Hello.
My suggestion: use existing config `trivyOperator.reportResourceLabels` for this.

**Reply by chen-keinan** [reply]
@mycodeself I agree with @louzadod this labels are added to report and also available for grouping via metrrics
